xml/System.Reflection.Emit/PersistedAssemblyBuilder.xml

```diff
@@ -266,9 +266,16 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates that the current assembly is a dynamic assembly.</summary>
+        <value>Always <see langword="true" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ An <xref:System.Reflection.Emit.PersistedAssemblyBuilder> object always represents a dynamic assembly.
+
+ ]]></format>
+        </remarks>
```

Comment:
nit: we are trying to avoid remarks as markdown wherever possible:

```suggestion
        <remarks>A <see cref="T:System.Reflection.Emit.PersistedAssemblyBuilder" /> object always represents a dynamic assembly.</remarks>
```